README.logging

```diff
@@ -1,3 +1,8 @@
+<!--
+SPDX-FileCopyrightText: Red Hat, Inc.
+SPDX-License-Identifier: GPL-2.0-or-later
+-->
+
```

Comment:
This is not markdown but using markdown looks good since this is likely to be converted to markdown. We can also use python/shell comments like you used in the commit template.